integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/support/SupportBase.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.core.base.support;
+
+import com.liferay.ide.core.base.action.EnvAction;
+
+import org.junit.rules.ExternalResource;
+
+/**
+ * @author Terry Jia
+ */
+public class SupportBase extends ExternalResource {
+
+	public SupportBase() {
+		envAction = EnvAction.getInstance();
```

Comment:
This should not use shared instance, instead it should just create a new object.